businessRules/ideas.js

```diff
@@ -1,59 +1,50 @@
-const { getAllIdeas, createIdea, updateIdea } = require('../models/ideas');
+const { getAllIdeas, createNewIdea, updateIdea } = require('../models/ideas');
+const {
+    validateHeader,
+    validateDescription,
+    validateAskedSum,
+    validateIdea,
+} = require('../validations/ideas');
+const { validateUserId } = require('../validations/users');
 
 const IDEA_STATUS = {
     pending: 'pending',
     accepted: 'accepted',
     rejected: 'rejected',
 };
 
-function getIdeas() {
-    return getAllIdeas();
-}
-
-function sortedByDonationSumIdeas() {
-    const ideasList = getIdeas();
+function sortIdeasByTotalDonationSum() {
+    const ideasList = getAllIdeas();
```

Comment:
I would rename `ideasList` go simply `ideas`

Reply:
The simplier, the better. Renamed.